distribution/docs/src/main/resources/content/_integrating/_endpoints/ftp-endpoint.adoc

```diff
@@ -2,49 +2,40 @@
 :type: endpoint
 :status: published
 :link: _ftp_endpoint
+:operations: ingest
+:url: \ftp://<FQDN>:8021/
```

Comment:
❓ What's the backslash for?

Reply:
to escape the URL from being an actual link that won't go anywhere